### PR TITLE
Cisco modules

### PR DESCRIPTION
**Pull request by francesco-noc**

### CODE REVIEW

File: module-types/Cisco/N7K-SUP2.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+manufacturer: Cisco
+model: N7K-SUP2
+part_number: N7K-SUP2
+console-ports:
+  - name: console{module}
+    type: rj-45
+  - name: com1{module}
+    type: rj-45
+interfaces:
+  - name: mgmt0
```
Comment (DanSheps) [inline module-types/Cisco/N7K-SUP2.yaml:11]: This will result in a name collision, please change this to somehow utilize the {module} to prevent this.

**Review by DanSheps — CHANGES_REQUESTED** [review]

**Review by DanSheps — APPROVED** [review]

**Review by DanSheps — CHANGES_REQUESTED** [review]
Please remove the pickle files from your commit and fix the merge conflicts

**Comment by danner26** [comment]
@francesco-noc one more just so you have it on your list

**Comment by francesco-noc** [comment]
Why are there always pickle files committed? I removed them before commit and push.
Am I missing some configuration?

**Comment by danner26** [comment]
> Why are there always pickle files committed? I removed them before commit and push. Am I missing some configuration?

You probably forked this too early and somehow got a different pickle file version. If you download the latest pickle files and add them in your commit, I think it'll fix the issue. That or start fresh from the latest master branch

File: module-types/Cisco/C9200-STACK.yaml
Comment (danner26) [inline module-types/Cisco/C9200-STACK.yaml:1]: Why does this have nothing else on it? Is it the stacking kit that inserts into the switch?

Comment (DanSheps) [reply module-types/Cisco/C9200-STACK.yaml:1]: it is a stack kit, I think it should have stack interfaces though

**Pull request closed without merging**